MovieDB/src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import  Search from './components/search';
+import MovieTileItem from './components/movie-tile-item';
+
+// create element examples
+const header = React.createElement('div', null, React.createElement('h1', null, 'This is a header'));
+const middle = React.createElement('div', null, 'here should be search bar');
+const footer = React.createElement('div', null, 'Footer');
+
+// create function component examples
+const listOfMovie = (movie) => {
```

Comment:
Naming is always a topic for a long discussion, so I  would just give my suggestions, but, of course, you can leave it as you wish: `listOfMovie ` -> `movies` or `movieList`

Reply:
Fixed. Renamed to movies.